lib/main.coffee

```diff
@@ -2,6 +2,9 @@ path = require 'path'
 fs = require 'fs-plus'
 _ = require 'underscore-plus'
 
+hasCommonJSPackageSemantics = (scopeName, filePath) ->
+  fs.isDirectorySync(filePath) and (scopeName.startsWith('source.js') or scopeName.startsWith('source.coffee'))
+
```

Comment:
Any reason use `startsWith('source.js')` instead of `scopeName is "source.js"`?

Reply:
Yeah. The babel-language and other JavaScript language modes that add jsx sub-namespace the scopeName: `"source.js.jsx"`

Reply:
thanks! understood.